tslint.json

```diff
@@ -48,7 +48,7 @@
       true,
       "tabs"
     ],
-    "interface-over-type-literal": true,
+    "interface-over-type-literal": false,
```

Comment:
I have a question. In other works, I changed the value of ʻinterface-over-type-literal` to `false`. Is there any reason?
@daybrush @WoodNeck

Reply:
@mixed
Interface can't be used as an event type.
In short, we can't use it like this:

```ts
interface MyInterface {
  prop: string;
}

new Component<{
  event0: MyInterface // Error!
}>
```

Reply:
@WoodNeck 
Thank you for your comments.
How are we currently using type and interface separately? Just in my opinion, events seem to fit the type.

Reply:
Before updating @egjs/component, it was just fine to use them as an interface.
As our linter prefers interface over object-based type, it was natural for us to use them as the interface.
Also, their main purpose was, like, for example of the Flicking it was for the jsdoc API document & internal usage.

Reply:
@WoodNeck 

The component extends the record (index signature). There is no index signature on the basis of `interface`. Index signature is possible if `type` is used.

Reply:
Does that means changing component itself can fix that problem?

Reply:
@WoodNeck 

No, it seems difficult to get the desired result.
Component `type` enhancement cannot use `interface`.

To give up the index signature(`{ [key: string]: any }`) method, you have to use `object` type.
However, it will be difficult to give up the index signature because of the types(function, object) supported by our component.